doc/source/index.rst

```diff
@@ -13,7 +13,7 @@ and MongoDB Community v3.6 or higher with `MongoDB Replication
 
 .. note::
 
-   |PBM| doesn't work on standalone |mongodb| instances. This is because |PBM| requires an :term:`oplog` to guarantee backup consistency. Oplog is available in clusters and replica sets only.
+   |PBM| doesn't work on standalone |mongodb| instances. This is because |PBM| requires an :term:`oplog <Oplog>` to guarantee backup consistency. Oplog is available in clusters and replica sets only.
```

Comment:
```suggestion
   |PBM| doesn't work on standalone |mongodb| instances. This is because |PBM| requires an :term:`oplog <Oplog>` to guarantee backup consistency. Replication must be enabled.
```
I think it must be confusing to talk about clusters, it is an concept orthogonal to replication.